GitUI/CommandsDialogs/FormFormatPatch.cs

```diff
@@ -179,6 +179,7 @@ private void FormatPatch_Click(object sender, EventArgs e)
             Close();
         }
 
+#pragma warning disable 618 // Mono marked SmtpClient obsolete
```

Comment:
https://github.com/mono/mono/commit/fa081c5ecd3b21353809d01ac2b08f7281f49ab0
this will probably have to be looked at some point